cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -96,6 +97,7 @@ def create_cluster(
     (Defaults to &#39;true&#39;.)
     :type rollback_on_failure: bool
     """
+    assert_node_executable(LOGGER)
```

Comment:
Is it possible to just add the `LOGGER` to the file that has the `assert_node_executable` function?

Reply:
Done